css/style.css

```diff
@@ -9,285 +14,609 @@ body {
+  left: 1407px;
+  top: 1665px;
+}
+
+/* Медиа запросы */
+
+@media (max-width: 3000px) {
+  .new-сollection__social {
+    width: 50%;
+    }
+}   
+
+@media (max-width: 1500px) {
+  .new-сollection__social {
+    width: 45%;
+    }
+} 
+
+@media (max-width: 1500px) {
+  .title {
+    top: 30%;
+    width: 20%;
+    font-size: 50px;
+    line-height: 52px;
+    left: 55%;
+    }
+} 
  
-        @media (max-width: 768px) { 
-            .new-сollection {
-               background: url(../img/girl2.jpg) no-repeat;
-               background-size: cover contain;      
-               background-position: 0 36%;        
-           }
-        }
-
-        @media (max-width: 768px) { 
-            .title {
-                font-size: 50px;
-                line-height: 51px;
-                position: absolute;
-                left: 5%;
-                top: 60%;
-                width: 297px;
-                height: 103px;
-           }
-        }
-
-        @media (max-width: 768px) { 
-            .galka {
-                position: absolute;
-                left: 170px;
-                top: 625px;
-           }
-        }
-       
\ No newline at end of file
+@media (max-width: 1500px) {
+  .text {
+    width: 35%;
+    left: 55%;
+    height: 60%;
+     }
+}
+ 
+@media (max-width: 1500px) {
+  .new-сollection__main-card {
+    width: 38%;
+    left: 55%;
+    margin-top: 1%;
+     }
+}
+     
+@media (max-width: 1500px) {
+  .girl {
+    width: 50%;
+    right: 60%;
+     }
+}
+
+@media (max-width: 1500px) {
+  .new-сollection__social {
+    width: 45.5%;
+   }
+}
+
+@media (max-width: 1300px) {
+  .new-сollection__social {
+    width: 47%;
+   }
+}
+
+@media (min-width: 1270px) {
+  .body {
+    margin: 0 auto;
+   }
+}
+
+@media (max-width: 1200px) { 
+  .new-сollection__social {
+    width: 45%;
+    }
+}
+
+@media (max-width: 1200px) { 
+  .girl {
+    width: 45%;
+    right: 60%;
+    }
+}
+ 
+@media (max-width: 1200px) { 
+  .new-сollection__main-button {
+    width: 35%;
+    font-size: 8px;
+     }
+}
+ 
+@media (max-width: 1200px) {
+  .new-сollection__main-link {
+    width: 50%;
+    margin-left: 3%;
+    font-size: 10px;
+    margin-right: -15%;
+     }
+}
+ 
+@media (max-width: 1200px) {
+  .new-сollection-arrow {
+    width: 10%;
+     } 
+}
+
+@media (max-width: 1200px) {
+  .new-сollection__social {
+    width: 47%;
+     }
+}
+
+@media (max-width: 961px) { 
+  .new-сollection__main-card {
+    margin-top: 5%;
+    }
+}
+
+@media (max-width: 961px) { 
+  .new-сollection__main-link {
+    margin-right: -5%;
+    }
+}
+
+@media (max-width: 961px) {
+  .new-collection__background {
+    max-width: 45%;
+    min-height: 900px;
+    }
+}
+
+@media (max-width: 961px) {
+  .logo {
+    margin-right: 20%;
+    }
+}
+
+@media (max-width: 961px) {
+  .new-сollection-main-link {
+    margin-left: 5%;
+   } 
+}
+
+@media (min-width: 800px) {
+  .logo-mobil,
+  .galka,
+  .new-collection__background_girl {
+    display: none;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected newline before "}" of a multi-line block (block-closing-brace-newline-before)](https://app.codacy.com/gh/WebHeroSchool/Love-metal_LOCO/pullRequest?prid=5449357)